MT_Test_ScenarioLoad.ahk

```diff
@@ -0,0 +1,110 @@
+﻿#Include <definitions>
```

Comment:
Issue #13 
Tested on my Local pc. Not yet tested in CI. logs are attached below.

[MT_Local_Test_ScenarioLoad_Logs.txt](https://github.com/NWLdragonite/MT_Automator/files/43593/MT_Local_Test_ScenarioLoad_Logs.txt)